string object_name
---
bool success
